quarkus/commons/src/main/java/org/apache/polaris/commons/QuarkusDatasourceSupplier.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.commons;
+
+import io.quarkus.arc.InstanceHandle;
+import java.util.List;
+import javax.sql.DataSource;
+import org.apache.polaris.extension.persistence.relational.jdbc.DatasourceSupplier;
+import org.apache.polaris.extension.persistence.relational.jdbc.RelationalJdbcConfiguration;
+
+public class QuarkusDatasourceSupplier implements DatasourceSupplier {
```

Comment:
It looks like we mixing this production code with test code in the same module, same package... This is not nice :thinking: 

WDYT about keeping this class under `quarkus/service` (and keeping test code as before)?

Reply:
I see may be i can put them under different directories, the thing i need this for both admin and service

Reply:
moved this code to `quarkus/common` and moved the tests to `quarkus/test-commons'